src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Utf8JsonWriterTests.cs

```diff
@@ -7747,6 +7893,24 @@ private static string GetDatesExpectedString(bool prettyPrint, string keyString,
             return Encoding.UTF8.GetString(ms.ToArray());
         }
 
+        private static void CompensateWhitespaces(bool prettyPrint, JsonTextWriter json, TextWriter streamWriter, int whitespaceCount = 1)
+        {
+            if (prettyPrint)
+            {
+                json.Flush();
+                streamWriter.Write(Enumerable.Repeat(' ', whitespaceCount).ToArray());
```

Comment:
what about `new string(' ', whitespaceCount)`

Reply:
Got it, thanks!